src/OpenTelemetry.Instrumentation.SqlClient/TracerProviderBuilderExtensions.cs

```diff
@@ -31,17 +31,19 @@ public static class TracerProviderBuilderExtensions
         /// </summary>
         /// <param name="builder"><see cref="TracerProviderBuilder"/> being configured.</param>
         /// <param name="configureSqlClientInstrumentationOptions">SqlClient configuration options.</param>
+        /// <param name="isEnabled">A delegate that filters events based on their name and up to two context objects.</param>
         /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
         public static TracerProviderBuilder AddSqlClientInstrumentation(
```

Comment:
We do not need to change this. Take a look at [ASP.NET Core]( https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Instrumentation.AspNetCore/AspNetCoreInstrumentation.cs ) for reference.

Reply:
Thank you @vishweshbankwar ! I'll change it. Here I have another question, like ASP.NET Core, do you mean we should fix the enabled events? And which events should be enabled? Like below?
```c#
private static readonly HashSet<string> DiagnosticSourceEvents = new()
        {
            "Microsoft.Data.SqlClient.WriteConnectionOpenBefore",
            "Microsoft.Data.SqlClient.WriteConnectionOpenAfter",
            "Microsoft.Data.SqlClient.WriteCommandBefore",
            "Microsoft.Data.SqlClient.WriteCommandAfter",
            "Microsoft.Data.SqlClient.WriteConnectionCloseBefore",
            "Microsoft.Data.SqlClient.WriteConnectionCloseAfter"
        };
```

Reply:
Also the enablement can be done by just updating [this](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Instrumentation.SqlClient/SqlClientInstrumentation.cs) file. We do not need to change public API.

Reply:
Got it!